merge.go

```diff
@@ -81,7 +90,13 @@ func (m *MergeSetConfig) ToString() (string, error) {
 	if m.Target != nil && (reflect.TypeOf(m.Target) == reflect.TypeOf(ParamString(""))) {
 		sb.WriteString(m.Name)
 		sb.WriteRune(' ')
-		sb.WriteString(EqualToOperator.String())
```

Comment:
You must fix the other `sb.WriteString(EqualToOperator.String())` below.

Reply:
22ebe08169fd05d8c917382172aa1128e36ae3a3